ServiceBoxWeb/src/main/java/com/infoshareacademy/controller/employee/ClientController.java

```diff
@@ -153,4 +154,32 @@ public String createUserAccount(@PathVariable("id") Long clientId, RedirectAttri
 
         return "redirect:/employee/clients";
     }
+
+    @GetMapping("/{id}/update")
```

Comment:
```suggestion
    @GetMapping("/client/{id}")
```
staraj się nie uzywac sugestywnych nazw endpointów, możesz dać np to co co dałem w przykładzie